docs/providers/aws/events/event-bridge.md

```diff
@@ -51,9 +53,9 @@ functions:
 
 The `eventBridge` event source will use the `default` event bus (the one AWS uses internally) when none is explicitly specified.
 
-The Serverless Framework will create the `eventBus` for you if you just provide a name for it. It will re-use an existing event bus if you provide an event bus `arn`.
+The Serverless Framework will create the `eventBus` for you if you just provide a name for it. It will re-use an existing event bus if you provide an event bus `arn`. Additionally, if you provide event bus with CloudFormation intrinsic function, it also won't be created.
```

Comment:
I think we can make it tad shorter, something as:

_The Serverless Framework will create the `eventBus` for you if you just provide a name for it. Otherwise when `arn` of existing bus is provided, Framework will attach to it._